package-lock.json

```diff
@@ -1,15074 +1,8 @@
 {
   "name": "@bcgsc-pori/graphkb-api",
-  "version": "3.13.4",
-  "lockfileVersion": 2,
+  "version": "3.13.5",
+  "lockfileVersion": 1,
```

Comment:
The lockfileversion should not have been downgraded. Did you use an older version of node here?

Reply:
Don't know why it changed, relly. I can't see myself knowingly using an older version of npm. Anyway, I reinstalled everything with npm 8.1.0, then runned into the same issue as here with the graphkb_loader https://github.com/bcgsc/pori_graphkb_loader/pull/102#issuecomment-1159134622 so fixed that too.

Reply:
Still says lockfileVersion 1. It should be 2, I think you will need to revert these commits and re-try ensuring you are using node 16

Reply:
It's not. It is lockfileVersion 2 now; where do you see lockfileVersion 1?

Reply:
weird, sorry about that. its showing as 1 on the diff in github. Perhaps that's a error in the display due to the size of the diff. I will checkout the branch and confirm it there

Reply:
you're right. manually looks ok

```git diff master..release/v3.13.5```